src/assets/i18n/en.json

```diff
@@ -31,6 +31,7 @@
     "SELECT_GEO_HAZARD": "Select geo hazard",
     "VIEW_WARNINGS": "View warnings from varsom.no",
     "TOGGLE_OBSERVATION": "Switch between simple and full observation",
+    "TOGGLE_OBSERVATION_EXTRA_INFO": "Toggle on to send observation there where you are - simple and fast. This is super useful warning data for other users.\n \n Toggle off to see all choices for sending an observation - just like before.",
```

Comment:
Det er ikke er gjort noen endringer på ordlyden her: 
_...send observation **there** where you are..._
Hva skjedde? Ble det glemt eller har du avklart med Solveig at det er slik det skal være?

Reply:
fikset nå :)